playground/nextjs/package.json

```diff
@@ -10,12 +10,12 @@
     "yalc:add": "yalc add -- @clerk/shared @clerk/types @clerk/backend @clerk/clerk-react @clerk/clerk-sdk-node @clerk/nextjs"
   },
   "dependencies": {
-    "@clerk/backend": "file:.yalc/@clerk/backend",
-    "@clerk/clerk-react": "file:.yalc/@clerk/clerk-react",
-    "@clerk/clerk-sdk-node": "file:.yalc/@clerk/clerk-sdk-node",
-    "@clerk/nextjs": "file:.yalc/@clerk/nextjs",
-    "@clerk/shared": "file:.yalc/@clerk/shared",
-    "@clerk/types": "file:.yalc/@clerk/types",
+    "@clerk/backend": "latest",
+    "@clerk/clerk-react": "latest",
+    "@clerk/clerk-sdk-node": "latest",
+    "@clerk/nextjs": "latest",
+    "@clerk/shared": "latest",
+    "@clerk/types": "latest",
```

Comment:
I think this will affect how the playground app is used locally. Which is mostly to check the local packages

Reply:
If you want to use Yalc you can run the corresponding yalc command.

Long-term if we use [secco](https://secco.lekoarts.de/) this doesn't matter anymore.